### PR TITLE
Danish localization

### PR DESCRIPTION
**Pull request by KunJakob**
Put together danish localization for the integration. Thanks for putting the effort into making this :)

### CODE REVIEW

File: custom_components/plex_assistant/localize.py [hunk 1]
```diff
@@ -911,6 +911,164 @@
...
+            "pre": [],
+            "post": [
+                "movies",
+                "movie",
+                "episodes",
+                "episode",
+                "tv",
+                "shows",
+                "show",
+                "of",
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1061]: Everything in ondeck section is untranslated.

File: custom_components/plex_assistant/localize.py [hunk 2]
```diff
@@ -911,6 +911,164 @@
...
+            "pre": [
+                'ste',
+                'en',
+                'je',
+                'de',
+                'te',
+            ],
+            "post": [
+                "number",
+                "of",
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1006]: The post section of the episode section is untranslated.

File: custom_components/plex_assistant/localize.py [hunk 3]
```diff
@@ -911,6 +911,164 @@
...
+            "keywords": [
+                "sæson",
+            ],
+            "pre": [
+                'st',
+                'nd',
+                'rd',
+                'th',
+                'the',
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:985]: Pre section of season is untranslated.

**Comment by maykar** [comment]
Thanks for this! I've marked some sections that appear to remain untranslated.

**Comment by maykar** [comment]
Let me know if you have any questions

**Comment by KunJakob** [comment]
Ah great! Sorry a bit unavailable at the moment, will get to this soonish

File: custom_components/plex_assistant/localize.py [hunk 4]
```diff
@@ -1044,6 +1044,164 @@
...
+            "post": [
+                "movies",
+                "movie",
+                "episodes",
+                "episode",
+                "tv",
+                "shows",
+                "show",
+                "of",
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1195]: whole ondeck section above needs translated

File: custom_components/plex_assistant/localize.py [hunk 5]
```diff
@@ -1044,6 +1044,164 @@
...
+                'ste',
+                'en',
+                'je',
+                'de',
+                'te',
+            ],
+            "post": [
+                "number",
+                "of",
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1140]: post section above needs translated

File: custom_components/plex_assistant/localize.py [hunk 6]
```diff
@@ -1044,6 +1044,164 @@
...
+            "keywords": [
+                "sæson",
+            ],
+            "pre": [
+                'st',
+                'nd',
+                'rd',
+                'th',
+                'the',
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1118]: pre section above needs translated

**Comment by maykar** [comment]
Remarked areas that need work as I pulled another translation in and it changed the line numbers.

**Comment by exetico** [comment]
Oh, damn. I've just translated all the things. Here's my take on it:
https://pastebin.com/pLSjTHuZ

**Comment by maykar** [comment]
If your translation is complete and ready to go, please make a PR for it. This one has been unfinished for a while now. No disrespect to KunJakob and I appreciate the effort, but I don't speak the language so I can't finish this one and don't like relying on google translate to finish these.

**Comment by exetico** [comment]
Okay. I've created #65 :+1:

**Pull request closed without merging**